deploy/db/sql/init/populate.sql

```diff
@@ -7,13 +7,13 @@ VALUES ('0x00Bd138aBD70e2F00903268F3Db08f2D25677C9e', 'user2@notary');
 
 -- Populate tokens table
 INSERT INTO notary.tokens (wallet, token) VALUES
-('0x9d65d6209bcd37f1f546315171b000663117d42f', 'omg'),
-('0x92fd99502914010942db94181061cdda1ce3b370', 'bat'),
-('0x084f2750c7eb6bb6e40cecfed5f54b198ab1b8c7', 'rep'),
-('0xbd10c47bfd673f7c0213c10e245bc7ffd4435c5b', 'btm'),
-('0x65327d02cf82f65331323a3cc9e908beca95d029', 'eos'),
-('0xba93c2539cea86ef4b59555167ac36debf59da4a', 'gnt'),
-('0x8e832894260c768d267fe405324f023b10f6bc3d', 'dgd'),
-('0x520924212c5ea068abf845e62bf2d531af8af635', 'mco'),
-('0x84d6edf34a7a1fb56520094ee671f39783199e7e', 'salt'),
-('0x220f2d5184127b01de2f0d0b1442eae96fff611b', 'icn');
+('0xd26114cd6EE289AccF82350c8d8487fedB8A0C07', 'omg'),
+('0x0d8775f648430679a709e98d2b0cb6250d2887ef', 'bat'),
+('0x1985365e9f78359a9B6AD760e32412f4a445E862', 'rep'),
+('0xcb97e65f07da24d46bcdd078ebebd7c6e6e3d750', 'btm'),
+('0x7b39940dbac110f1227d37c395675def270afcd7', 'eos'),
```

Comment:
It looks like this address belongs to EOS Gold, not EOS. EOS can be found at https://etherscan.io/token/0x86fa049857e0209aa7d9e616f7eb3b3b78ecfdb0.